### ISSUE TITLE
UX on shorty terminal - focus goes out of view on `azd init` on version 1.3.0

### ISSUE THREAD

**Issue by vhvb1989** [Azure/azure-dev#2720]
Running `azd init` from a short terminal is now confusing because the focused selection is hidden in the top.  See next picture, it is a VSCode terminal at the bottom of VSCode, after running `azd init`:

![image](https://github.com/Azure/azure-dev/assets/24213737/69898022-e52e-4905-adcc-d3c316d97cd5)

The only way to get the expected output is by making the terminal to be half size of the screen:

![image](https://github.com/Azure/azure-dev/assets/24213737/51e2e963-3ba5-4531-9bdb-fc6bd4526fe6)

The same behavior is seen on Windows Powershell, using a small terminal:

![image](https://github.com/Azure/azure-dev/assets/24213737/19c0bcdd-77c8-4233-93c1-6ae5c63ee07c)

Note:  Using the arrow keys up and down only makes it worst, as it makes azd to repaint the output and duplicates all the output. See this by using the up/down arrow keys and then use the mouse to scroll up in the terminal.  A weir output is observed:

![image](https://github.com/Azure/azure-dev/assets/24213737/5e6e3e73-27ef-4b53-9670-a90e0477a43b)

I suspect that, the longer the list of templates is, the bigger the terminal will need to be to avoid this issue.

**Comment by savannahostrowski** [comment]
@Austinauth for his input here

**Comment by Austinauth** [comment]
I agree with the analysis here. I've observed this personally and seen others struggle with this as well. Until I have some dedicated time to look at this pattern, I recommend reverting back to the simple, single line list select pattern (no double spaces, max of 7 items in the list). The GitHub path is a nice touch, but we may need to remove it for the time being. 

I'll make note to come back to this pattern during the Design Guide work in Germanium. In the meantime if anyone has suggestions feel free to propose them here.

**Comment by savannahostrowski** [comment]
I would prefer we don't revert the change quite yet. I don't see this as blocking right now so I'd prefer we keep this as is until we can spend some additional time assessing the UX here.

Would love to consider this fix alongside some of the other pieces of feedback we've gotten for the template list via https://github.com/Azure/azure-dev/issues/2732 and https://github.com/Azure/azure-dev/issues/2731.

**Comment by kristenwomack** [comment]
closing this issue because we no longer have this issue 

<img width="1203" height="541" alt="Image" src="https://github.com/user-attachments/assets/2a35a55c-63e1-4743-9d95-976bcac1ac09" />
